src/data/set/lattice.lean

```diff
@@ -67,7 +67,7 @@ instance : has_Sup (set α) := ⟨sUnion⟩
 /-- Intersection of a set of sets. -/
 def sInter (S : set (set α)) : set α := Inf S
 
-prefix `⋂₀`:110 := sInter
+prefix `⋂₀ `:110 := sInter
```

Comment:
Does this match the `sUnion` notation?

Reply:
No but `sUnion` is in core. I think that we should add spaces to both.

Reply:
Agreed! Can you then just leave a little TODO note, unless you already have a PR open to core, just so that we don't forget?

Reply:
https://github.com/leanprover-community/lean/pull/675

Reply:
Perfect!